src/file.ts

```diff
@@ -0,0 +1,20 @@
+import * as vscode from 'vscode'
+
+const isPackageJson = (document: vscode.TextDocument) => {
+  // Is checking both slashes necessary? Test on linux and mac.
+  return document.fileName.endsWith('\\package.json') || document.fileName.endsWith('/package.json')
+}
+
+const isPnpmWorkspace = (document: vscode.TextDocument) => {
+  return document.fileName.endsWith('pnpm-workspace.yaml')
```

Comment:
Could this use the `path` nodejs package and [`basename`](https://nodejs.org/api/path.html#pathbasenamepath-suffix), then just check to see if they are equal?